### PR TITLE
Changed Reporting documentation

### PR DESCRIPTION
**Pull request by SvetlanaUgrinov**
Changed Reporting documentation

### CODE REVIEW

File: docs/admin/services/reporting-service/config-connector-console.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ The Connector service has several values that need to be configured for it to fu
 | configStore:table | The name of the table to manage connector service settings in, e.g. _ReportingConfig_. |
 | configStore:connection | The connection string to use for connecting to a database to store the settings table in. DBO rights are needed to create the table. |
 | global:ssrsEndpointUrl | The SSRS reporting service endpoint URL noted during SSRS setup with _/ReportService2010.asmx_ appended, e.g. _https://example.com/ReportServer/ReportService2010.asmx_. |
-| global:ssrsRootFolderPath | The SSRS root folder path for this connector instance as noted during SSRS worksapce template setup, e.g. _SymbioReporting_.  |
 | global:ssrsUserName | The name of a user which has management access to the SSRS root folder path given above. |
 | global:ssrsPassword | The password of the user named above. |
+| global:ssrsRootFolderPath | The SSRS root folder path for this connector instance as noted during SSRS worksapce template setup, e.g. _SymbioReporting_.  |
 | global:templateName | The name of the template folder in the root path given above as noted during SSRS workspace template setup, e.g. __Template_. |
```
Comment (Gerrit-P-und-Z) [inline docs/admin/services/reporting-service/config-connector-console.md:23]:
```suggestion
| global:templateName | The name of the template folder in the root path given above as noted during SSRS workspace template setup, e.g. _Template_. |

```

**Review by Gerrit-P-und-Z — DISMISSED** [review]

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+To set up the **Reporting Connector** you will need either an IIS installation with a current version of the ASP.NET Core Hosting package installed or an Azure App Service ready for deployment. In both cases the service will need access to Symbio, SSRS/Power BI, and the ODS database. It must also be reachable by the server side component of Symbio.
+
+## Installation
+
+#### Reporting Connector on IIS
+1. Create a directory to host the files of the connector service, e.g. _C:\SymbioServices\Reporting\Connector_
+2. Copy all files and folders of the service into this directory
+3. In IIS Manager:
+    1. Create a new Application pool (No Managed Code/Integrated) and assign a user account with appropriate rights to the created directory
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:14]: I guess, I will describe the "appropriate" rights in the FAQ - for logging we need R/W access at least...

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    2. Create a new Application using the created pool and selecting the created directory
+    3. Note the resulting URL (composed of website binding URL and the selected alias/path of the new application)
+4. Configure the Connector by editing its appsettings.config files 
+5. Test connection to service by navigating to the noted URL
+
+#### Reporting Connector as Azure App Service
+
+1. Deploy the service package
+2. Ensure that the resulting web.config is not automatically set to the version V2 of the AspNetCoreModule
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:23]: Is V2 still a problem or is it fixed in the meantime on Azure?

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Configuration
+
+
+The Connector service has several values that need to be configured for it to function properly. These values can be set in its appsettings.config file (IIS) or the app service settings of Azure.
+
+| Setting | Meaning |
+| ------- | ------- |
+| security:authToken | The freely defineable token string that Symbio needs to provide to be allowed access to this connector service instance. |
+| configStore:type | Either _SqlServer_ or _AzureTableStorage_; the type of database server to connect to. |
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:37]: I am not sure but Azure Table Storage has been removed, hasn't it?

File: docs/admin/services/reporting-service/config-symbio.md [hunk 5]
```diff
@@ -22,18 +22,20 @@ In Symbio follow these steps to create and configure a report pool:
 | Service endpoint url | Enter the URL of the connector service as noted during its setup. |
 | Authentication token | Enter the security:authToken that was defined in the connector service's settings. |
 | Administrative connection string | Enter a connection string to the ODS database that allows full access to the database. |
-| Rea-only connection string | Enter a connection string the the ODS database that allows read-only access to the database. |
+| Read-only connection string | Enter a connection string the the ODS database that allows read-only access to the database. |
+| SSRS root folder path | The SSRS root folder path for this connector instance as noted during SSRS worksapce template setup, e.g. SymbioReporting. |
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/config-symbio.md:26]: Why do we need to provide the root folder and the template folder path in both locations: here and in the appsettings of the connector? What takes precedence?

File: docs/admin/services/reporting-service/config-symbio.md [hunk 6]
```diff
@@ -22,18 +22,20 @@ In Symbio follow these steps to create and configure a report pool:
 | Service endpoint url | Enter the URL of the connector service as noted during its setup. |
 | Authentication token | Enter the security:authToken that was defined in the connector service's settings. |
 | Administrative connection string | Enter a connection string to the ODS database that allows full access to the database. |
-| Rea-only connection string | Enter a connection string the the ODS database that allows read-only access to the database. |
+| Read-only connection string | Enter a connection string the the ODS database that allows read-only access to the database. |
+| SSRS root folder path | The SSRS root folder path for this connector instance as noted during SSRS worksapce template setup, e.g. SymbioReporting. |
+| Report template folder | The name of the template folder in the root path given above as noted during SSRS workspace template setup, e.g. _Template. |
 | SSRS enabled | Mark this checkbox if you want to support SSRS reports with this report pool. |
 | Power BI enabled | Mark this checkbox if you want to support Power BI reports with this report pool. |
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/config-symbio.md:29]: I guess, we should remove this option as it is not implemented (yet) and therefore misleading.

File: docs/admin/services/reporting-service/config-symbio.md [hunk 7]
```diff
@@ -22,18 +22,20 @@ In Symbio follow these steps to create and configure a report pool:
...
 ### Connecting a storage to a report pool
 
 In Symbio follow these steps to connect a storage to a report pool:
 
-1. Login as an admin user to the _collection_ where the storage is located
+1. Login as an admin user to the _Collection_ where the storage is located
 2. Switch to _Storages_
 3. Select the storage to connect to a report pool
-4. Select the desired report pool from the drop down
+4. Select the desired _Report pool_ from the drop down
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/config-symbio.md:38]: I suggest being consistent in the casing of terms, so use "Report pool" in the sentences above, too.

File: docs/admin/services/reporting-service/reporting-console.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Reporting Console 
+
+## Requirements
+
+To set up the **Reporting Console** you will need a machine with a current version of the .NET Core runtime installed. This machine needs access to Symbio and the ODS database.
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-console.md:5]: No connection to the reporting connector needed to access the actual configuration?

**Review by JochenLinnemann — DISMISSED** [review]

File: docs/admin/services/reporting-service/config-symbio.md (hunk 5 above)
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/config-symbio.md:26]: We need at least one. In ReportingConfigStore.cs we have:

 public string GetSsrsRootFolderPath(Guid? reportPoolId)
        {
            return _configStore.GetAsync($"{reportPoolId.GetValueOrDefault()}.{SSRS_ROOT_FOLDER_PATH}").Result
                              ?? _configuration.GetValue("global:ssrsRootFolderPath", string.Empty);
        }

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
+# Reporting Connector 
+
+
+## Requirements
+
+To set up the **Reporting Connector** you will need either an IIS installation with a current version of the ASP.NET Core Hosting package installed or an Azure App Service ready for deployment. In both cases the service will need access to Symbio, SSRS/Power BI, and the ODS database. It must also be reachable by the server side component of Symbio.
```
[the remark this answers is not in the document]
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/reporting-connector.md]: Removed Power BI

File: docs/admin/services/reporting-service/config-symbio.md (hunk 6 above)
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/config-symbio.md]: Removed Power BI

File: docs/admin/services/reporting-service/config-symbio.md (hunk 7 above)
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/config-symbio.md]: Changed

File: docs/admin/services/reporting-service/ssrs-and-reports.md [hunk 10]
```diff
@@ -1,42 +1,44 @@
...
+
+Please secure your Report Service with an HTTPS endpoint and choose a valid and accepted certificate. Remember that this URL must be available to browsers in your organization (for displaying reports).
+
+## Workspace Template
+
+The [Reporting Connector](reporting-connector.md) is expecting a template directory in SSRS that will be used to create workspaces for different report pools configured in Symbio.
+
+It is recommended that you create one root folder in SSRS per Reporting Connector that will use that SSRS instance. In the image below this root folder has been named _SymbioReporting_. If you create a root folder (which is optional), please note its name (or path using forward slashes if you create a deeper nested structure).
+
+In your Reporting Connector's root folder create a template folder and note its name. In the example below this folder has been named _Template_. For each report pool in Symbio, the connector will create a copy of the template folder and assign it the ID of the report pool as its name.
```
[the remark this answers is not in the document]
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/ssrs-and-reports.md]: Changed

File: docs/admin/services/reporting-service/overview.md [hunk 11]
```diff
@@ -19,10 +19,10 @@ To use Symbio Reporting, the following requirements must be fullfilled:
 
 - Symbio set up and running
 - SQL Server or Azure SQL set up and running
-- Either [SSRS](deploy-ssrs.md) (requires SQL server) or Power BI set up and running
+- Either [SSRS](ssrs-and-reports.md) (requires SQL server) or Power BI set up and running
```
[the remark this answers is not in the document]
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/overview.md]: Removed Power BI

File: docs/admin/services/reporting-service/reporting-console.md (hunk 8 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/reporting-console.md:5]: It needs a connection to the reporting connector, too!

File: docs/admin/services/reporting-service/config-connector-console.md (hunk 1 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/config-connector-console.md]: Should not be changed as in the examples the template folder starts with an underline character.
If __Template_ is not easy to read, change it to *_Template*

Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/config-connector-console.md]: i.e. use * instead of _ for italics.

**Comment by CPatPundZ** [comment]
Newer examples are already using non_underscore version, at least where I am working on…

File: docs/admin/services/reporting-service/reporting-console.md (hunk 8 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/reporting-console.md:5]: Or at least the storage you want to work on must be connected to a reporting connector instance.

File: docs/admin/services/reporting-service/config-connector-console.md (hunk 1 above)
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/config-connector-console.md]: Changed, tnx

File: docs/admin/services/reporting-service/config-symbio.md [hunk 12]
```diff
@@ -1,8 +1,8 @@
-# Symbio Reporting - Configuration
+# Symbio Reporting
 
 ## Requirements
 
-Reporting Connector & Console are already [set up](deploy-connector-console.md).
+Reporting [Connector](reporting-connector.md) & [Console](reporting-console.md) are already set up.
```
[the remark this answers is not in the document]
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/config-symbio.md]: Changed

File: docs/admin/services/reporting-service/reporting-console.md (hunk 8 above)
Comment (SvetlanaUgrinov) [reply docs/admin/services/reporting-service/reporting-console.md]: Changed

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 3 above)
Comment (CPatPundZ) [reply docs/admin/services/reporting-service/reporting-connector.md:23]: There is no issue with V2 anymore, especially when .NET Core 3.1 is used, V2 module is mandatory! So, this is closeable.

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 4 above)
Comment (CPatPundZ) [reply docs/admin/services/reporting-service/reporting-connector.md:37]: Renamed to CosmosDB as it seems. But "Azure Table Storage" is still a term in Azure which points to CosmosDB. Removed was a nuget package with the name AzureTableStorage.

File: docs/admin/services/reporting-service/reporting-console.md (hunk 8 above)
Comment (CPatPundZ) [reply docs/admin/services/reporting-service/reporting-console.md]: Yes, correct is that the storage needs a configured and valid report pool, and the reporting connector configured for this report pool must be active and valid.
The console application itself does not talk with the connector directly, only with Symbio REST API.

**Review by CPatPundZ — DISMISSED** [review]
Renamed Template consistently without underscore.

**Review by Gerrit-P-und-Z — DISMISSED** [review]

**Review by Gerrit-P-und-Z — APPROVED** [review]

**Review by CPatPundZ — APPROVED** [review]

**Pull request merged**